From 0f6ba599691fecf898daf445f49c932a1413bb73 Mon Sep 17 00:00:00 2001
From: Mauro Dreissig <mukadr@gmail.com>
Date: Wed, 1 Jul 2020 18:44:20 -0300
Subject: [PATCH] staging: rtl8712: base_types: Remove unused macros
Git-commit: 0f6ba599691fecf898daf445f49c932a1413bb73
Patch-mainline: v5.9-rc1
References: jsc#SLE-13430

Those #define's are not used anywhere, get rid of them.

Signed-off-by: Mauro Dreissig <mukadr@gmail.com>
Link: https://lore.kernel.org/r/20200701214420.5566-3-mukadr@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/staging/rtl8712/basic_types.h | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/staging/rtl8712/basic_types.h b/drivers/staging/rtl8712/basic_types.h
index b8ecd92ca9d4..aecded87dd4c 100644
--- a/drivers/staging/rtl8712/basic_types.h
+++ b/drivers/staging/rtl8712/basic_types.h
@@ -14,12 +14,8 @@
 #ifndef __BASIC_TYPES_H__
 #define __BASIC_TYPES_H__
 
-#define SUCCESS	0
-#define FAIL	(-1)
-
 #include <linux/types.h>
 
-#define SIZE_T __kernel_size_t
 #define sint signed int
 
 /* Should we extend this to be host_addr_t and target_addr_t for case:
@@ -28,8 +24,5 @@
  */
 #define addr_t unsigned long
 
-#define MEM_ALIGNMENT_OFFSET	(sizeof(SIZE_T))
-#define MEM_ALIGNMENT_PADDING	(sizeof(SIZE_T) - 1)
-
 #endif /*__BASIC_TYPES_H__*/
 
-- 
2.16.4

